pkgs/build-support/zig/build-zig-package/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, zig }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  buildPhase = attrs.buildPhase or ''
+    runHook preBuild
+    export HOME=$(mktemp -d)
+    zig build -Drelease-safe -Dcpu=baseline $zigBuildFlags
```

Comment:
Is `zigBuildFlags` an environment variable?

Reply:
I believe so. Just use it like so.

```nix
zigBuildFlags = "-Dexample-flag" ;
```